### ISSUE TITLE
Can't start up logstash

### ISSUE THREAD

**Issue by fjspang** [elastic/logstash#9675]
Please post all product and debugging questions on our [forum](https://discuss.elastic.co/c/logstash). Your questions will reach our wider community members there, and if we confirm that there is a bug, then we can open a new issue here.

For security vulnerabilities please only send reports to security@elastic.co.
See https://www.elastic.co/community/security for more information.

Logstash Plugins are located in a different organization: https://github.com/logstash-plugins. For bugs on specific Logstash plugins, for example, if Redis Output has a defect, please open it in the respective Redis Output repository. 

For all general issues, please provide the following details for fast resolution:

- Version:
- Operating System:
- Config File (if you have sensitive info, please remove it):
- Sample Data:
- Steps to Reproduce:

**Comment by fjspang** [comment]
Hi
I saw a similar question, but it does not apply to my issue 100%.
I downloaded logstash:
wget https://artifacts.elastic.co/downloads/logstash/logstash-6.2.4.tar.gz
tar -xvf logstash-6.2.4.tar.gz

Then to run it, I did the following:
~/logstash-6.2.4/bin> ./logstash -e 'input { stdin { } } output { stdout {} }'
The response was: Can't start up: not enough memory
I verified I had enough memory by typing the top command (4 GB free out of 8 GB)

My linux distribution is: OpenSuse Leap 42.3 (installed in my second hard disk).
Java distribution is:
---------------------
:~> java -version
openjdk version "1.8.0_161"
OpenJDK Runtime Environment (IcedTea 3.7.0) (build 1.8.0_161-b12 suse-21.1-x86_64)
OpenJDK 64-Bit Server VM (build 25.161-b12, mixed mode)
---------------------


Apart from what I'm showing I did nothing else. Thanks for any help.
Francisco
